jobs/cloud_controller_ng/templates/post-backup-unlock.sh.erb

```diff
@@ -17,6 +17,10 @@ source /var/vcap/packages/capi_utils/syslog_utils.sh
   <% (1..(p("cc.jobs.local.number_of_workers"))).each do |index| %>
   monit_start_job cloud_controller_worker_local_<%= index %>
   <% end %>
-  wait_for_server_to_become_healthy <%= "localhost:#{p("cc.external_port")}/healthz" %> <%= p("cc.post_bbr_healthcheck_timeout_in_seconds") %>
-  sleep 30
+  set +e
```

Comment:
I wonder if this script is even set to `-e` because #L13 might `return 0` after setting `+e`